evalml/tests/data_checks_tests/test_target_leakage_data_check.py

```diff
@@ -173,32 +173,81 @@ def test_target_leakage_none():
 def test_target_leakage_types():
     leakage_check = TargetLeakageDataCheck(pct_corr_threshold=0.8)
 
-    y = pd.Series([1, 0, 1, 1])
+    y = pd.Series(
+        [1, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1]
```

Comment:
nit: you could replace this with the `[1, 0, 1] * X` format if you wanted!

Reply:
Done!